webapp/src/main/webapp/vue-apps/matrix/components/chatButton.vue

```diff
@@ -105,6 +107,7 @@
         this.openRoom(urlParams.get('roomId'));
       }
 
+      this.checkCanCreatePrivateRooms();
```

Comment:
Could you move this logic inside the quick create room drawer and avoid having this extra call  in the chat button ?